modules/installation-user-infra-machines-iso.adoc

```diff
@@ -59,10 +59,19 @@ coreos.inst=yes
 coreos.inst.install_dev=sda <1>
 coreos.inst.image_url=<bare_metal_image_URL> <2>
 coreos.inst.ignition_url=http://example.com/config.ign <3>
+ip=<dhcp or static IP address> <4>
 ----
 <1> Specify the block device of the system to install to.
 <2> Specify the URL of the RAW image that you uploaded to your server.
 <3> Specify the URL of the Ignition config file for this machine type.
+<4> Set `ip=dhcp` or set an individual static IP address (ip=) and DNS server (nameserver=) on each node.
```

Comment:
Some advanced notes that may be unnecesary:

- You can specify multiple `ip=` lines if your system has multiple network interfaces.
- You can mix DHCP and static IP configurations when your system has multiple network interfaces, but you must specify the interface used.  (i.e. `ip=dhcp:enp1s0 ip=10.10.10.2::10.10.10.254:255.255.255.0:core0.example.com:enp1s1:none`)
- You can specify multiple instances of `nameserver=` to provide alternate DNS servers.

Reply:
@miabbott, if you use multiple DNS servers, do you use a comma-separated list to specify them?

Reply:
Nope, just separate `nameserver=` entries, for example:

`ip=10.10.10.2::10.10.10.254:255.255.255.0:core0.example.com:enp2s0:none nameserver=4.4.4.41 nameserver=1.1.1.1 nameserver=8.8.8.8`

Reply:
Thank you! I'm going to merge Chris's PR and open a follow-up to capture your notes.